spec/turn_spec.rb

```diff
@@ -0,0 +1,56 @@
+require './lib/turn'
+require './lib/card'
+require 'rspec'
+require 'pry'
+
+RSpec.describe Card do
+  it 'exists' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+
+    expect(card).to be_instance_of(Card)
+  end
+end
```

Comment:
Since we've already tested this in the `card_spec` we don't also need to include it here, even though `Turn` stores a `card`. It is okay to assume that the testing of `Card` in `card_spec` is sufficient coverage for `Card` here, too.